app/views/events/_event.html.erb

```diff
@@ -119,4 +141,42 @@
   </div>
 <% end %>
 
+<p>Here is the crucial info</p>
+<% if @event.theme_image_url %>
+  <%= image_tag @event.theme_image_url, style: 'width: 100%; height: 200px' %>
+<% end %>
+
+<div class="row mt-3">
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-12">
+    <p><b><%= @event.name %>: <%= @event.theme %></b></p>
+    <%= simple_format(@event.theme_details) %>
+  </div>
+
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-12">
+    <p>Where: <%= @event.location %></p>
+    <p>Map: <%= link_to 'Google Maps Location', @event.maps_location_url %></p>
+    <% if @event.eventbrite_event.eventbrite_event.start.local.nil? %>
+      <p>When: <%= simple_format(@event.event_timings) %></p>
+    <% else %>
+      <p>When: <%= Date.parse(@event.eventbrite_event.eventbrite_event.start.local).strftime("%A %d %B %Y") %> <%= simple_format(@event.event_timings) %></p>
```

Comment:
Does the eventbrtie gem not parse this for us? Is there a possibility of it being nil? This feels like quite a lot of logic for the view. Maybe we move it into event.rb, so we can call something like `@event.eventbrite_start_time.strftime("%A %d %B %Y")`?

Reply:
What I mean is add something like (I haven't tested this) this method to event.rb:

```ruby
def eventbrite_start_time
 Date.parse(eventbrite_event.eventbrite_event.start.local)
end
```

and then you can call that in the view much more tidily, and if we need to add any more logic to it, it keeps tidy,

Reply:
Done!